src/main/webapp/components/NoteGridCard.vue

```diff
@@ -1,36 +1,63 @@
 <template>
-  <div class="note-grid-card" @click="$emit('click')">
-    <div class="thumbnail" :style="thumbnail"></div>
+  <v-col>
+    <v-card class="note-grid-card" @click="$emit('click')"
+            width="208px">
+      <v-img :src="thumbnailSrc"
+             height="180px"
+             position="top">
+        <v-icon>{{ isGNote ? "mdi-google-drive" : "mdi-pdf-box" }}</v-icon>             
+      </v-img>
 
-    <div class="metadata">
-      <div class="note-title">{{title}}</div>
+      <v-divider></v-divider>
 
-      <div class="info-row">
-        <p>{{school}} &bull; {{course}}</p>
+      <v-card-title>
+        <v-list-item-title class="d-block text-truncate">{{title}}</v-list-item-title>
+      </v-card-title>
 
-        <img src="assets/gdrive.webp">
-        {{dateCreated}}
-        <div class="rating-box">
-          {{numFavorites}}
-          <span class="star">&star;</span>          
-        </div>
-      </div>
-    </div>
-  </div>
+      <v-card-subtitle>
+        <v-list-item-subtitle>{{ course }} • {{ school }}</v-list-item-subtitle>
+      </v-card-subtitle>
+
+      <v-card-text>
+        <v-row no-gutters>
+          <v-col>
+            {{dateCreated}}
+          </v-col>
+          <v-col col="1">
+            <v-badge overlap class="ma-1">
+              <template v-slot:badge>{{numFavorites}}</template>
+              <v-icon :color="favorited && 'yellow'">mdi-star</v-icon>
+            </v-badge>
+            <v-badge overlap class="ma-1">
+              <template v-slot:badge>{{numDownloads}}</template>
+              <v-icon>mdi-download</v-icon>
+            </v-badge>
+          </v-col>
+        </v-row>
+      </v-card-text>
+    </v-card>
+  </v-col>
 </template>
 
 <script>
   module.exports = {
     props: {
       thumbnailSrc: String,
+      id: Number,
       title: String,
       dateCreated: String,
       school: String,
       course: String,
       labels: Array,
       numDownloads: Number,
       numFavorites: Number,
-      isFavorited: Boolean
+      isFavorited: Boolean,
+    },
+    data: function() {
+      return {
+        favorited: this.isFavorited,
+        isGNote: true,
```

Comment:
Do you think that this should be a computed value instead of in data?

Reply:
Makes, sense did that and added a TODO for computing it :)